src/Components/Bookpage/BookDetails/ClaimForm/index.js

```diff
@@ -60,59 +59,49 @@ function ClaimForm({ title, claimed }) {
             .then((res) => res.json())
             .then((data) => {
                 if (data.message !== `Book ${id} is already claimed`) {
```

Comment:
What if it is already claimed? Worth displaying a message

Reply:
done